setup.py

```diff
@@ -671,6 +671,41 @@ def add_multiarch_paths(self):
         finally:
             os.unlink(tmpfile)
 
+    def add_wrcc_search_dirs(self):
+        # add library search path by wr-cc, the compiler wrapper
+
+        def convert_mixed_path(path):
+            # convert path like C:\folder1\folder2/folder3/folder4
+            # to msys style /c/folder1/folder2/folder3/folder4
+            drive = path[0].lower()
+            left = path[2:].replace("\\", "/")
+            return "/" + drive + left
+
+        cc = sysconfig.get_config_var('CC')
+        tmpfile = os.path.join(self.build_temp, 'wrccpaths')
+        if not os.path.exists(self.build_temp):
+            os.makedirs(self.build_temp)
+        ret = os.system('%s --print-search-dirs >%s' % (cc, tmpfile))
+        try:
+            if ret >> 8 == 0:
+                with open(tmpfile) as fp:
+                    for line in fp.readlines():
+                        if line.startswith("libraries"):
+                            # On Windows building machine, VxWorks does
+                            # cross builds under msys2 environment.
+                            sep=";" if sys.platform == "msys" else ":"
```

Comment:
I suggest to rename the variable to pathsep, to mimick os.path.pathsep name. Also, please add spaces around the = operator (PEP 8):

```suggestion
                            pathsep = (";" if sys.platform == "msys" else ":")
```

Also, please define it outside the loop (don't define it at each iteration). Maybe before the try block.

Reply:
This has been moved into sub-function add_search_path().